playbooks/cluster-up.yaml

```diff
@@ -33,76 +40,98 @@
       groups: ec2hosts
     loop: "{{ ec2.instances }}"
 
+  - copy:
+      src: "{{ template_ocp_inventory_file }}"
+      dest: "{{ src_ocp_inventory_file }}"
+
+  - replace:
+      path: "{{ src_ocp_inventory_file }}"
+      regexp: '^(openshift_cloudprovider_aws_access_key=)'
+      replace: "\\1{{ lookup('env', 'AWS_ACCESS_KEY_ID') }}"
+
+  - replace:
+      path: "{{ src_ocp_inventory_file }}"
+      regexp: '^(openshift_cloudprovider_aws_secret_key=)'
+      replace: "\\1{{ lookup('env', 'AWS_SECRET_ACCESS_KEY') }}"
+
+  - replace:
+      path: "{{ src_ocp_inventory_file }}"
+      regexp: '^(openshift_clusterid=)'
+      replace: "\\1{{ tag_name }}"
+
+# Pre-deployment
 - hosts: ec2hosts
   name: wait for ssh to come up
   remote_user: "{{ ssh_login }}"
   gather_facts: false
-  
   tasks:
   - wait_for_connection:
       delay: 20
       timeout: 120 
 
-# Everything after this can be modified to our needs. 
+# OCP deployment
 - hosts: ec2hosts
   name: configuration
-  remote_user:  "{{ ssh_login }}"
+  remote_user: "{{ ssh_login }}"
   gather_facts: True
   become: yes
-  vars:
-    known_hosts_path: /root/.ssh/known_hosts
-    key_path: /root/.ssh/ocp_key
   environment:
-    OCP_CLUSTER_ID: "{{ tag_name }}"
+    ANSIBLE_HOST_KEY_CHECKING: false
```

Comment:
should there be any check to make sure this var and the 2 below are not blank?

Reply:
`ANSIBLE_HOST_KEY_CHECKING` defaults to true, so it's never blank.  We're overriding it here.